src/app/zap-templates/templates/chip/CHIPClusters-src.zapt

```diff
@@ -35,7 +38,9 @@ CHIP_ERROR {{asCamelCased clusterName false}}Cluster::{{asCamelCased name false}
     uint8_t argSeqNumber = 0;
 {{/first}}
     // {{asCamelCased label}}: {{asCamelCased type}}
-{{#if (isString type)}}
+{{#if (isByteString type)}}
+    ReturnErrorOnFailure(writer.Put(TLV::ContextTag(argSeqNumber++), {{asCamelCased label}}));
```

Comment:
Why do we need to add `{{#if (isByteString type)}}` ? This seems to be the same code as the `{{else}}` case